payments/billing-sdk/src/main/java/com/appcoins/billing/sdk/sku_details/ProductSerializer.kt

```diff
@@ -2,15 +2,12 @@ package com.appcoins.billing.sdk.sku_details
 
 import com.appcoins.payments.arch.ProductInfoData
 import com.google.gson.Gson
-import javax.inject.Inject
-import javax.inject.Singleton
 
 internal interface ProductSerializer {
   fun serialize(products: List<ProductInfoData>): ArrayList<String>
 }
 
-@Singleton
-internal class ProductSerializerImpl @Inject constructor() : ProductSerializer {
+internal class ProductSerializerImpl internal constructor() : ProductSerializer {
```

Comment:
The internal class has internal constructor by default so we don't need to specify the `internal` modifier for it explicitly